README.md

```diff
@@ -18,3 +18,29 @@ Todo:
 
 - contribute support to grpc-java Netty implementation to provide async start/shutdown
 - worker integration ?
+
+## Plugin installation
+
+To use Reactor-gRPC with the protobuf-maven-plugin, add a [custom protoc plugin configuration section](https://www.xolstice.org/protobuf-maven-plugin/examples/protoc-plugin.html).
```

Comment:
please change : it is not "Reactor-gRPC"

Reply:
can you handle this small change ?

Reply:
I don't know how to modify this pr, that's why I opened another one on August 28

Reply:
Do you want to change the target version on first line of readme to 4.0.0-snapshot?

Reply:
can you do a PR on the branch 15-support-for-rxgrpc in this repo ? when I merged commits in this branch they will be added to this PR

Reply:
Yes this is just what i did on 28 august. PR #47

Reply:
I missed that sorry

can you make a new PR rebase on the branch because it says _This branch cannot be rebased due to conflicts_ ?

thanks

Reply:
PR #50

Reply:
thanks!